packages/@sfcc-core/apiconfig/src/index.ts

```diff
@@ -17,15 +16,13 @@ Object.keys(process.env).forEach(key => {
     }
 });
 
-export default class APIConfig {
+export class APIConfig {
+    logger = core.logger;
+    _config: { [key: string]: string };
```

Comment:
This is the syntax for defining an object with string keys and values. In other words, `{ 1: 2 }` would be invalid